src/resources/LibraryStyles.css

```diff
@@ -397,7 +398,6 @@ input {
 .SearchResultItemContainer .ItemIcon {
     padding: 2px 8px;
     width: 40px;
-    height: 40px;
```

Comment:
Yeah, this "workaround" looks better! And can we change the search result icon size to 32px instead of 40px? I guess this size will go better since we decided to make everything in the LibraryUI look smaller in order to match the rest of the Dynamo UI.